apps/web/src/app/arrangementer/components/AttendanceCard/AttendanceCard.tsx

```diff
@@ -32,9 +32,9 @@ import { TicketButton } from "./TicketButton"
 import { ViewAttendeesButton } from "./ViewAttendeesButton"
 
 const getQueuePosition = (
-  attendee: Attendee | undefined,
-  attendees: Attendee[] | undefined,
-  attendablePool: AttendancePool | undefined
+  attendee: Attendee | null | undefined,
+  attendees: Attendee[] | null | undefined,
+  attendablePool: AttendancePool | null | undefined
```

Comment:
Is this actually a tri-state or not? Under what circumstances can the attendance be:

1. There
2. Null
3. Undefined

It sounds like it should be either T|null or T|undefined, not T|undefined|null